src/lib/interfaces/request.ts

```diff
@@ -2,7 +2,10 @@ interface HttpRequest {
   method: 'PUT' | 'POST' | 'DELETE' | 'GET'
   url: string
   headers?: object
-  data?: object
+  data?: {
+    sys?: object,
```

Comment:
For which kind of requests we need to send `sys`? 🤔

Reply:
That is not supposed to be there, thanks!